src/adagrams.js

```diff
@@ -1,8 +1,153 @@
 const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+    drawLetters() {
+        // Implement this method for wave 1
+        let poolOfLetters = [];
+        const distroOfLetters = {
+            A: 9,
+            B: 2,
+            C: 2,
+            D: 4,
+            E: 12,
+            F: 2,
+            G: 3,
+            H: 2,
+            I: 9,
+            J: 1,
+            K: 1,
+            L: 4,
+            M: 2,
+            M: 2,
```

Comment:
minor, but, watch out! You have a duplicate `M` here. I saw this because VS Code gave me a small warning saying there was a duplicate key!